module/updatable_configs/validation/require_approvals_for_seal_construction.go

```diff
@@ -0,0 +1,35 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/module/defaults"
+)
+
+// ValidateRequireApprovals validates the given value against the default of other values.
+func ValidateRequireApprovals(requiredApprovalsForSealConstruction uint) error {
+	// NOTE: validating against the default value. Even though we take in new values from flag, but
+	// in practice we never use this flag, because it would need a coordination of all nodes updating
+	// the flag, which will most likely done by updating a new image with a new default value.
+	return validateRequireApprovals(
+		requiredApprovalsForSealConstruction,
+		defaults.DefaultRequiredApprovalsForSealValidation,
+		// chunks.DefaultChunkAssignmentAlpha,
+		3,
+	)
+}
+
+func validateRequireApprovals(requiredApprovalsForSealConstruction uint, requiredApprovalsForSealVerification uint, chunkAlpha uint) error {
+	// We need to ensure `requiredApprovalsForSealVerification <= requiredApprovalsForSealConstruction <= chunkAlpha`
+	if requiredApprovalsForSealVerification > requiredApprovalsForSealConstruction {
+		return fmt.Errorf("invalid consensus parameters, requiredApprovalsForSealVerification (%v)  > requiredApprovalsForSealConstruction (%v)",
+			requiredApprovalsForSealVerification, requiredApprovalsForSealConstruction)
+	}
+
+	if requiredApprovalsForSealConstruction > chunkAlpha {
```

Comment:
technically, the largest valid threshold is `requiredApprovalsForSealConstruction <= 2/3 * chunkAlpha` Otherwise, the system is not robust against byzantine nodes. Imagine we set `requiredApprovalsForSealConstruction = chunkAlpha`. Then a single byzantine verifier could halt the entire protocol by not approving.

Reply:
Could we add a TODO for this, and a note for why it's acceptable to leave as-is for now?